tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -9,7 +9,6 @@
 class Search(Base):
 
     name = 'Browser'
-    manifest_url = "app://search.gaiamobile.org/manifest.webapp"
```

Comment:
I removed it because we use the class name now here https://github.com/mozilla-b2g/gaia/pull/31259#discussion-diff-37986457R155 , so we don't need to define it